api/prometheus/v1/api_test.go

```diff
@@ -872,3 +876,90 @@ func TestAPIClientDo(t *testing.T) {
 
 	}
 }
+
+func TestSamplePair(t *testing.T) {
```

Comment:
need a comment to explain why do we need this test or what is it testing or a more descriptive name like: `TestSamplesJsonMarshalingCorrectness`